lua/dashboard/session.lua

```diff
@@ -35,33 +29,81 @@ function session.session_save(name)
   end
 
   local file_name = name == nil and project_name() or name
-  local file_path = db.session_directory .. '/' .. file_name .. '.vim'
+  local file_path = db.session_directory .. path_sep .. file_name .. '.vim'
   api.nvim_command('mksession! ' .. fn.fnameescape(file_path))
   vim.v.this_session = file_path
-  vim.notify('Session ' .. file_name .. ' is now persistent')
+
+  if db.session_verbose then
+    vim.notify('Session ' .. file_name .. ' is now persistent')
+  else
+    vim.notify('This session is now persistent')
+  end
+  db.session_cp_name = project_name()
```

Comment:
why public this variable. it not a config option.

Reply:
I made it a db field so that it is a persistent value and won't be reset on every `require("dashboard.session")` which would defeat the purpose of the variable.